### PR TITLE
[FIX] web: prevent rest time while drag and drop on month mode

### PR DESCRIPTION
**Pull request by sabr-odoo**
Specification:
The time should not be reset when dragging an event in the month view.
    
Observed behavior:
Time is reset when dragging an event in month mode.
   
Task-3432065

### CODE REVIEW

**Review by mcm-odoo — DISMISSED** [review]
Hello, the fix seems wrong. We lose the time info when the event is dropped not when we prepare the event to be displayed.
Check the function `fcEventToRecord`, we only receive the dates without time in month view. In month view, we could merge the date received from the move and the time from the existing record.

**Comment by Polymorphe57** [comment]
Hello @sabr-odoo . Apparently, you have a better fix on another branch. Could you please push it here?

**Comment by sabr-odoo** [comment]
Hello @mcm-odoo @Polymorphe57 ,

I have made the changes. Could you please take a look at it?

Thank you🙂

File: addons/web/static/src/views/calendar/calendar_common/calendar_common_renderer.js [hunk 1]
```diff
@@ -294,7 +294,20 @@ export class CalendarCommonRenderer extends Component {
             }
         }
         if (id) {
-            res.id = this.props.model.records[id].id;
+            const existingRecord = this.props.model.records[id];
+            if (this.props.model.scale === "month" && existingRecord) {
```
Comment (Polymorphe57) [inline addons/web/static/src/views/calendar/calendar_common/calendar_common_renderer.js:298]: either existingRecord can be removed or there is a problem below with
res.id = existingRecord.id;

**Comment by Polymorphe57** [comment]
@robodoo r+

**Pull request closed without merging**